imagelab/ImgProvider.java

```diff
@@ -490,5 +497,25 @@ public void showPixNew(String name) {
         dImage1.changeImage(this,"Second Pass");
         System.out.println("ImgProvider:showSlow: Second Pass");
     }
+    
+        public void removeContainer(){
+        dis.remove();
+    }
+    
+    public int[] getPix(){
+        return pix;
+    }
+    
+    public int getPixWidth(){
+        return pixwidth;
+    }
+    
+    public int getPixHeight(){
+        return pixheight;
+    }
+    
+    public DisplayImage getDis(){
+        return dis;
+    }
```

Comment:
This entire section of the public API requires documentation.  At the very least, there needs to be Javadoc comments for the published API.

Reply:
I have added JavaDocs to these getters. They are available in the latest PR.